File: src/codes/Users_Manual/link/simulator_0.cpp
//~#include<autd3.hpp>
#include "autd3/link/simulator.hpp"

//~int main() {
//~auto autd = autd3::ControllerBuilder({autd3::AUTD3(autd3::Point3::origin())})
//~                .open(
autd3::link::Simulator::builder("127.0.0.1:8080")
    //~                    );
    //~return 0; }

File: src/codes/Users_Manual/link/simulator_0.cs
~using AUTD3Sharp;
~using System.Net;
using AUTD3Sharp.Link;

~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~var autd = Controller.Builder([new AUTD3(Point3.Origin)]).Open(
Simulator.Builder(new IPEndPoint(IPAddress.Parse("127.0.0.1"), 8080))
~);